### PR TITLE
Network configs

### PR DESCRIPTION
**Pull request by nkryuchkov**
Changed `BeaconNetwork`'s underlying type from `string` to `struct` with network parameters

### CODE REVIEW

File: types/network_config.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	SlotDuration:           12 * time.Second,
+	SlotsPerEpoch:          32,
+	CapellaForkEpoch:       162304, // Goerli taken from https://github.com/ethereum/execution-specs/blob/37a8f892341eb000e56e962a051a87e05a2e4443/network-upgrades/mainnet-upgrades/shanghai.md?plain=1#L18
+	Domain:                 V3Testnet,
+	DepositContractAddress: "0xff50ed3d0ec03ac01d4c79aad74928bff48a7b2b",
+	GenesisValidatorsRoot:  "043db0d9a83813551ee2f33450d23797757d430911a9320530ad8a0eabc43efb",
+	GenesisEpoch:           152834, // TODO: another value?
+}
+
+func GetBeaconTestNetwork() BeaconNetwork {
```
Comment (nkryuchkov) [inline types/network_config.go:40]: remove

**Comment by nkryuchkov** [comment]
Roots are fixed in https://github.com/nkryuchkov/ssv-spec/pull/3

**Comment by nkryuchkov** [comment]
> Roots are fixed in [nkryuchkov#3](https://github.com/nkryuchkov/ssv-spec/pull/3)

https://github.com/nkryuchkov/ssv-spec/pull/3 was merged into this PR

File: types/network.go [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+	Bootnodes            []string
+}
+
+type ETHParams struct {
+	NetworkName      string
+	SlotDuration     time.Duration
+	SlotsPerEpoch    uint64
+	MinGenesisTime   uint64
+	CapellaForkEpoch spec.Epoch
+}
```
[the remark this answers is not in the document]
Comment (y0sher) [reply types/network.go:33]: Initially we wanted to support custom network parameters and for this we had to extract all the constants from the spec code into structs. If this (as seems) is not a priority anymore, we can just use the existing string type from the spec.

**Comment by moshe-blox** [comment]
Created an issue to re-do this change properly: https://github.com/bloxapp/ssv-spec/issues/262

**Pull request closed without merging**